doc/release-notes.md

```diff
@@ -10,6 +10,30 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new string-valued option, `-allowdeprecated` has been introduced to make it possible
+  for users to turn off deprecated methods and features in the RPC api. In the case that
+  this parameter is not provided, all currently deprecated RPC methods and features remain
+  enabled, but this added CLI parameter makes it now possible to explicitly select a
+  subset of these deprecated features to enable, or to disable all deprecated features
+  entirely by providing the string `"none"` as the argument to this parameter. To explicitly
+  enable only a specific set of deprecated features, use `-allowdeprecated=<flag1>,...,<flagN>` 
```

Comment:
Instead of comma-separated single arg, use `mapMultiArgs` so that we instead pass `-allowdeprecated` multiple times, once for each flag (as is done for every other multi-arg option to `zcashd`).